test/column/cellClass.test.ts

```diff
@@ -0,0 +1,65 @@
+import { screen } from '@testing-library/react';
+import { Column } from '../../src';
+import { setup } from '../utils';
+
+interface Row {
+  id: number;
+}
+
+const rows: readonly Row[] = [{ id: 0 }, { id: 1 }];
+
+function getAllCellClassNames() {
+  return screen.getAllByRole('gridcell').map(row => row.className);
+}
+
+test('cellClass is undefined', () => {
+  const columns: readonly Column<Row>[] = [{
+    key: 'id',
+    name: 'ID',
+    cellClass: undefined
+  }];
+  setup({ columns, rows });
+  expect(getAllCellClassNames()).toStrictEqual([
```

Comment:
Wdyt about using https://github.com/testing-library/jest-dom#tohaveclass?

https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-the-wrong-assertion

Reply:
Sounds good. I've had this blog post bookmarked for a while now, I should get to it at some point.

Reply:
![image](https://user-images.githubusercontent.com/567105/98104704-b709e380-1e8e-11eb-92ca-27e74925ad9d.png)
The error messages can be buggy 🤔